x-pack/legacy/plugins/reporting/export_types/png/server/execute_job/index.test.js

```diff
@@ -14,63 +13,68 @@ import { LevelLogger } from '../../../../server/lib';
 
 jest.mock('../lib/generate_png', () => ({ generatePngObservableFactory: jest.fn() }));
 
+let mockReporting;
```

Comment:
Noticing we have to setup a lot of mocks for certain modules for the config object. Do you think it'd be easier/better to have these getters called "a layer up" to simplify testing/mocks? That way these modules only deal with primitive values and don't have to call getters?

Reply:
I think there are certain areas at the top-level where it is good to pass the entire config/getter object. But I went through the changes and found some leaf areas that can just get the config data passed in: ad8269c8b1